freeciv-web/src/main/webapp/javascript/fc_types.js

```diff
@@ -27,6 +27,7 @@ var TRI_MAYBE = 2;
 var MAX_NUM_ITEMS = 200;
 var MAX_NUM_ADVANCES = 250;
 var MAX_NUM_UNITS = 250;
+var MAX_NUM_BUILDINGS = 250;
```

Comment:
Not really a bug, as it's not used in the client (neither this nor B_LAST), but it's 200 there, like MAX_NUM_ITEMS.